app/code/Magento/Shipping/Controller/Adminhtml/Order/Shipment/Save.php

```diff
@@ -86,7 +91,7 @@ protected function _saveShipment($shipment)
      * Save shipment
      * We can save only new shipment. Existing shipments are not editable
      *
-     * @return void
+     * @return \Magento\Backend\Model\View\Result\Redirect
```

Comment:
In one case there will be result Jason, so I prefer to write there result interface. It’s really minor, I can fix it myself